subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/parameters/BuildProgressListenerAdapter.java

```diff
@@ -290,6 +317,16 @@ private ProjectConfigurationProgressEvent toProjectConfigurationProgressEvent(In
         }
     }
 
+    private TransformProgressEvent toTransformProgressEvent(InternalProgressEvent event, InternalTransformDescriptor descriptor) {
```

Comment:
I wonder if all the public facing parts should also be called `Transformation...`. What is your perception of the current naming?

Reply:
From `DependencyHandler`:

```java
    /**
     * Register an artifact transformation.
     *
     * @see org.gradle.api.artifacts.transform.ArtifactTransform
     * @since 3.5
     */
    void registerTransform(Action<? super VariantTransform> registrationAction);
```

I think this shows our own inconsistency in naming these things. However, all public-facing classes I'm aware of use `Transform` instead of `Transformation`. Thus, I think we should stick with that for the TAPI as well. WDYT?

Reply:
We can stick with `Transform` in the public API. I guess it would add more confusion if we introduced `Transformation` in the public API now.